.github/workflows/release_workflow.yml

```diff
@@ -4,6 +4,8 @@ on:
   release:
     types: [published]
 
+  workflow_dispatch:
```

Comment:
This line allows us to manually run workflows. We need to add this because the release workflow currently only triggers when releasing new versions. However, we want to run it manually to fix the v1.3.0 docs.